src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working");

```

Comment:
Similarly, the stop message should be more informative. Instead of just 'Excavator stopped working', it could be 'Excavator has finished digging and is now off'.